keel-titus-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/titus/cluster/TitusClusterSpec.kt

```diff
@@ -52,28 +55,40 @@ data class TitusClusterSpec(
   @JsonInclude(JsonInclude.Include.NON_EMPTY)
   val overrides: Map<String, TitusServerGroupSpec> = emptyMap(),
   @JsonIgnore
-  override val deliveryArtifact: DeliveryArtifact? = null,
-  @JsonIgnore
+  val containerProvider: ContainerProvider,
+  @JsonProperty(access = Access.WRITE_ONLY)
+  override val artifactName: String? = null,
+  @JsonProperty(access = Access.WRITE_ONLY)
+  override val artifactType: ArtifactType? = ArtifactType.docker,
```

Comment:
Artifact type is now assumed here, since this spec can only really be associated with that type.

Reply:
Same question - is there anything making sure of this?